package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@pathwaysml/engine",
+  "author": {
+    "name": "Pathways",
+    "url": "https://pathways.spongeass.dev"
```

Comment:
i call getting started with this cause i have a fun homepage idea (can i use server-first like astro or should i use a fullstack framework like nextjs for better future web chat / log viewing features)